pygmt/src/grdtrack.py

```diff
@@ -43,7 +45,9 @@
     w="wrap",
 )
 @kwargs_to_strings(R="sequence", S="sequence", i="sequence_comma", o="sequence_comma")
-def grdtrack(grid, points=None, newcolname=None, outfile=None, **kwargs):
+def grdtrack(
+    grid, points=None, output_type="pandas", outfile=None, newcolname=None, **kwargs
```

Comment:
Why don't `newcolname` has a default value like `z`?

Reply:
OK, we can discuss if `newcolname` should have a default value when more module wrappers have the same parameter.